js_test.go

```diff
@@ -1230,3 +1230,34 @@ func TestJetStreamStreamInfoWithSubjectDetails(t *testing.T) {
 		t.Fatalf("expected 0 subjects details from StreamInfo, but got %d instead", len(result.State.Subjects))
 	}
 }
+
+func StreamNameBySubject(t *testing.T) {
```

Comment:
This test will not be executed without `Test` prefix.
```suggestion
func TestStreamNameBySubject(t *testing.T) {
```